csvtotable/convert.py

```diff
@@ -88,17 +77,16 @@ def freeze_js(html):
         return html
 
     # Reverse regex matches to replace match string with respective JS content
-    reversed_matches = reversed([m for m in matches])
-    for match in reversed_matches:
+    for match in reversed(matches):
```

Comment:
Since `matches` is not a sequence you can't directly pass it to `reversed` function.
```
Traceback (most recent call last):
  File "/Users/vivek/.virtualenvs/csvtotable/bin/csvtotable", line 9, in <module>
    load_entry_point('csvtotable==1.1.0', 'console_scripts', 'csvtotable')()
  File "/Users/vivek/.virtualenvs/csvtotable/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Users/vivek/.virtualenvs/csvtotable/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/vivek/.virtualenvs/csvtotable/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/vivek/.virtualenvs/csvtotable/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/vivek/Documents/projects/python/csvtotable/csvtotable/cli.py", line 30, in cli
    delimiter=delimiter, quotechar=quotechar)
  File "/Users/vivek/Documents/projects/python/csvtotable/csvtotable/convert.py", line 53, in convert
    js_freezed_html = freeze_js(html)
  File "/Users/vivek/Documents/projects/python/csvtotable/csvtotable/convert.py", line 80, in freeze_js
    for match in reversed(matches):
TypeError: argument to reversed() must be a sequence
```